Common/UnitDefinitions/CoefficientOfThermalExpansion.json

```diff
@@ -0,0 +1,46 @@
+﻿{
+  "Name": "CoefficientOfThermalExpansion",
+  "BaseUnit": "InverseKelvin",
+  "XmlDoc": "A unit that represents a fractional change in size in response to a change in temperature.",
+  "BaseDimensions": {
+    "Θ": -1
+  },
+  "Units": [
+    {
+      "SingularName": "InverseKelvin",
+      "PluralName": "InverseKelvin",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "K⁻¹" ]
```

Comment:
Chrome on mobile is not reading this symbol right. Is the file utf8 encoded?

Reply:
Odd. I see the same on my mobile. Looks fine on the web and in Visual Studio. Yes it is UTF-8. It's a superscript -1.